src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -243,19 +622,52 @@
 		"sourcePath" : "log.sagebase.org/fileUploadRecords/",
 		"tableDescriptor": {
 			"name": "fileuploadrecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"user_id": "bigint",
-				"project_id": "bigint",
-				"file_handle_id": "bigint",
-				"association_object_id": "bigint",
-				"association_object_type": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"record_date": "date"
+			"description" : "The table contain upload records for FileEntity and TableEntity.The events are recorded only after the file is successfully uploaded.",
+			"columns":[{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when file upload event is pushed to the queue, after upload of a file."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The id of the user who uploaded the file."
 			},
+			{
+				"name": "project_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the project where the uploaded entity resides. Applicable only for FileEntity and TableEntity."
+			},
+			{
+				"name": "file_handle_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the file handle."
+			},
+			{
+				"name": "association_object_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the Synapse object ( with out prefix syn) that wraps the file."
+			},
+			{
+				"name": "association_object_type",
+				"type" : "string",
+				"comment" : "The type of the Synapse object that wraps the file."
+			},
+			{
+				"name": "stack",
+				"type" : "string",
+				"comment" : "The stack (prod, dev) on which the file was uploaded."
```

Comment:
Test stack?

Reply:
we have only 2 stack dev and prod. All test/user defined stacks are on dev. Staging and prod is Prod stack.